app/containers/RoomHeader/RoomHeader.stories.tsx

```
@@ -100,3 +100,60 @@ export const Themes = () => (
 		<ThemeStory theme='black' />
 	</>
 );
+
+const futureExpiry = '2030-06-15T19:00:00.000Z';
+
+export const DM_Status = () => {
+	const hourLaterExpiry = new Date(Date.now() + 3600000).toISOString();
```

ESLint

[error] Error: Cannot call impure function during render

`Date.now` is an impure function. Calling an impure function can produce unstable results that update unpredictably when the component happens to re-render. (https://react.dev/reference/rules/components-and-hooks-must-be-pure#components-and-hooks-must-be-idempotent).

/home/jailuser/git/app/containers/RoomHeader/RoomHeader.stories.tsx:107:35
  105 |
  106 | export const DM_Status = () => {
> 107 | 	const hourLaterExpiry = new Date(Date.now() + 3600000).toISOString();
      | 	                                 ^^^^^^^^^^ Cannot call impure function
  108 | 	const expiredExpiry = new Date(Date.now() - 3600000).toISOString();
  109 | 	return (
  110 | 	<>

(react-hooks/purity)